tests/nfv/vsperf_installation.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: openvswitch installation and CLI test
+#
+#   This test does the following
+#    - Installs openvswitch (dpdk if DPDK==1)
+#    - Starts the systemd service unit
+#    - Executes a few basic openvswitch commands
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.de>
+
+use base "consoletest";
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    zypper_call('in git', timeout => 200);
+
+    # Clone repository
+    assert_script_run "git clone https://gerrit.opnfv.org/gerrit/vswitchperf";
+
+    # Checkout the patch supporting SLE15 (to be removed once the patch is merged upstream)
+    assert_script_run "cd vswitchperf/systems";
+    assert_script_run "git fetch https://gerrit.opnfv.org/gerrit/vswitchperf refs/changes/17/48017/3 && git checkout FETCH_HEAD";
```

Comment:
Defenetly need to extracted to a variable, also from git perspective can this "refs/changes..." be replaced with branch name ? You can define special branch where you will commit whatever changes you need and from test code we will not see some scary numbers which no one except you understand :)

Reply:
I will remove this line as soon as https://gerrit.opnfv.org/gerrit/#/c/48017/ is merged, then we won´t have scary numbers